src/app/AllowCustomSchemesDialog.qml

```diff
@@ -0,0 +1,61 @@
+/*
```

Comment:
Hmm. Functionally, this dialog is fine. But I wonder if we can move it around and change the text to make it clearer. @cibersheep, do you have any thoughts?

![screenshot20190915_174200521](https://user-images.githubusercontent.com/21966173/64928869-89259380-d7e3-11e9-907f-823f346f0d45.png)

Reply:
Yep. I think is a bit confusing because of the structure.
I got there doing:
 - going to t.me[blbalba]
- Allow/Block dialog t.me. Tapped allow
- Custom URL schemes dialog saying «t.me is blocked»

In this Dialog, what about something like this?:

*Custom URL Scheme*
This site is trying to ¿open? the URL scheme: `tg:[blablab]`

This will try to dispatch this url to a system app if possible. Do you want to proceed?

Always allow URL schemes from t.me []

[allow]
[Cancel]

Reply:
So, first I get 

![imatge](https://user-images.githubusercontent.com/6640041/64931833-55765800-d83b-11e9-8cef-2ec37ef3fffd.png)

then 

![imatge](https://user-images.githubusercontent.com/6640041/64931840-6626ce00-d83b-11e9-9e2e-599cd9d769a0.png)

Reply:
For reference, here are some examples of the same dialog from Chromium and Firefox:

![image](https://user-images.githubusercontent.com/21966173/64971377-3a244080-d86d-11e9-9034-653ba799a8c8.png)

![Screenshot at 2019-09-16 10-30-23](https://user-images.githubusercontent.com/21966173/64971392-427c7b80-d86d-11e9-8399-9ccd7461899e.png)

Of course we can't select the application to launch, nor would the browser know which application is going to be launched.

Reply:
Firefox message seems more understandable (+1 FF)